torch_geometric/explain/metrics.py

```diff
@@ -0,0 +1,46 @@
+from typing import Tuple
+
+import torch
+from torchmetrics import AUROC, ROC
+
+from torch_geometric.explain import Explanation
+
+
+def groundtruth_metrics(
+        explanation: Explanation,
+        groundtruth: Explanation) -> Tuple[float, float, float, float, float]:
+    """accuracy_scores: Compute accuracy scores when
```

Comment:
Describe thresholding behaviour in the docstring, i.e. *>0 thresholding* for TP, and FP.

Reply:
Also explain the order of returned metrics